src/ecalc/libraries/libecalc/common/libecalc/input/validation_errors.py

```diff
@@ -116,6 +117,7 @@ def __init__(
     ):
         errors = validation_error.errors()
 
+        component_name = data[EcalcYamlKeywords.name]
```

Comment:
I don't think it is guaranteed that name is available, so we should check if name is available then add it to the message if it is.